.env.sample

```
@@ -0,0 +1,8 @@
+DATABASE_URL="mysql://username:password@127.0.0.1:3306/database_name"
```

```suggestion
DATABASE_URL="rdbms://username:password@localhost:db_port/database_name"
```
.env.sample에는 최대한 중의적이게 작성되는 것이 좋습니다! 어떠한 rdbms를 쓰는지도, rdbms port number도 모두 마찬가지입니다 :)